akvo/rsr/tests/rest/test_project_hierarchy.py

```diff
@@ -65,14 +73,14 @@ def setUp(self):
 
     def test_project_tree_check(self):
         # roots
-        self.assertEqual(self.p1.parents_all().count(), 0)
-        self.assertEqual(self.p4.parents_all().count(), 0)
-        self.assertEqual(self.p5.parents_all().count(), 0)
+        self.assertEqual(self.p1.ancestors(with_self=False).count(), 0)
+        self.assertEqual(self.p4.ancestors(with_self=False).count(), 0)
+        self.assertEqual(self.p5.ancestors(with_self=False).count(), 0)
 
         # tree
-        self.assertIn(self.p1, self.p2.parents_all())
-        self.assertIn(self.p2, self.p3.parents_all())
-        self.assertIn(self.p5, self.p6.parents_all())
+        self.assertIn(self.p1, self.p2.ancestors(with_self=False))
+        self.assertIn(self.p2, self.p3.ancestors(with_self=False))
+        self.assertIn(self.p5, self.p6.ancestors(with_self=False))
```

Comment:
See my reply on `akvo/rsr/views/utils.py`

Reply:
Fixed in 0bdf7a0377119604493b898af3b1b212d54eaee5